### ISSUE TITLE
Exporting text containing "[[ ]]" (link format in org-mode)

### ISSUE THREAD

**Issue by jpuerj** [louietan/anki-editor#24]
I'm trying to use anki-editor with an addon called Cloze Overlapper, basically I need to write a field which contains this syntax: [[oc1::text_to_hide]]. However, org-mode thinks it is a link and the export fails. Is there any workaround for this? Thank you for this awesome package.

**Comment by louietan** [comment]
Wrapping it into a verbatim by `~[[oc1::text_to_hide]]~` or a code by `=[[oc1::text_to_hide]]=` should keep it as it is, but not sure if the result could be recognized by Cloze Overlapper.

**Comment by jpuerj** [comment]
Thanks louietan. As you suspected, the result could not be recognized, however I found a good solution. Maybe it can be useful for someone, so I will explain. 

Cloze Overlapper generate fields Text1, Text2, etc; whose content is the previous content without occlusion and the next item to be hide. I create Yasnippets snippets with trigger key 'o1' which creates:

> {{c1::${1:cloze1}}}
*** Text2
$1
$0

This way I mirror the content of first cloze to the second, the way Cloze Overlapper works, I created snippets for the next levels to 'o2', 'o3', etc...

Thank you, Anki Editor Team!
